src/components/UpcomingBirthdays/UpcomingBirthdays.js

```diff
@@ -1,9 +1,40 @@
 import './UpcomingBirthdays.css';
+import { useSelector } from 'react-redux';
 
 const UpcomingBirthdays = () => {
+  const friends = useSelector(state => state.friends);
+  const upcomingBirthdays = friends.filter(friend => {
+    const today = new Date();
+    const monthFromToday = new Date(today.getTime() + 30*24*60*60*1000);
+    const currentDate = today.toISOString().split('T')[0];
+    const currentMonthFromToday = monthFromToday.toISOString().split('T')[0];
```

Comment:
Nice job with this complicated date manipulation here. If we run into more date stuff, it might pay to use a package like date.js to help us out